packages/dds/tree/README.md

```diff
@@ -0,0 +1,3 @@
+# @fluidframework/tree
```

Comment:
Should it be in the experimental namespace if it's not read for prime time? I prefer the explicit break that using experimental implies. Consumers will be forced to change their imports (or do some other voodoo) once you're prod ready. Or at least that your api surface is stable enough to warrant a non-experimental designation.

Reply:
@DLehenbauer, do I recall correctly that you were the one who asked me to put the tree DDS here?

Reply:
@tylerbutler - I'd normally agree, but in this case we want to send a strong signal to customers of the existing '@experimental/sharedtree' and '@experimental/propertytree' DDSes that this is the production codebase they want to align with.

(I do like @CraigMacomber's recommendation of marking it private until we're ready for beta, though.  Esp. as there will be some code movement shortly to put the right layering in place.)

Reply:
@DLehenbauer @yann-achard-MS Sounds good! Thanks for putting thought into this.